### ISSUE TITLE
[Bug] Token 1.3 update missing

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-ios#916]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

any

### OS Version

any

### App version

0.18.0

### App build number

1755794658

### Tokens libraries versions embeded in the app

OUDS core 1.5.0

### App build type

beta

### Describe the issue

The following tokens have issues:
V 1.3.0
* Opacity
  * disables

* Dimension > Space
  * spacePaddingBlocks2xs -> missing
  * spacePaddingBlocks3xs -> missing
  * spacePaddingBlocks4xs -> missing
  * spacePaddingBlocksmd -> missing
  * spacePaddingBlockslg -> missing
  * spacePaddingBlocksxl -> missing
  * spacePaddingBlocks2xl -> missing
  * spacePaddingBlocks3xl -> missing
  * spacePaddingBlocks4xl -> missing
  * spacePaddingBlocksMedium -> does not exist anymore
  * spacePaddingBlocksTall -> does not exist anymore
  * spacePaddingBlocksTaller -> does not exist anymore
  * spacePaddingBlocksTallest -> does not exist anymore

* Color
  * OverlayModal -> does not have the right value
  * contentOnActionSelected -> missing
  * contentStatusPositive -> light color and dark colors are inverted
  * contentStatusInfo -> light color and dark colors are inverted
  * contentStatusWarning -> light color and dark colors are swapped
  * contentStatusNegative -> light color and dark colors are swapped
  * actionSupportDisabled -> missing
  * actionVisitedLight -> light color and dark colors are swapped
  * 

### Expected Behavior

make sure the tokens are present in the token description pages

### Accessibility settings

na

**Comment by pylapp** [comment]
Duplicate of https://github.com/Orange-OpenSource/ouds-ios/issues/915

**Comment by B3nz01d** [comment]
Closed during OUDS sprint # 20
